usaspending_api/common/serializers.py

```diff
@@ -113,5 +119,28 @@ def setup_eager_loading(cls, queryset, prefix=""):
 
 class AggregateSerializer(serializers.Serializer):
 
+    def __init__(self, *args, **kwargs):
+        super(AggregateSerializer, self).__init__(*args, **kwargs)
+
+        include_fields = None
+
+        req = self.context.get('req')
```

Comment:
Not sure, but - this harvesting of parameters from `req` and/or `request` might be a good candidate for pulling out into a helper function, both because (1) it's repeated in serializers.py, and (2) that would be a good chance to give it a clarifying name; as it stands I had to squint at it for a minute wondering what's going on.